src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -222,32 +235,51 @@
                 WITHDRAW_CLAIM,
                 DISCONTINUE_CLAIM,
                 CASE_PROCEEDS_IN_CASEMAN,
-                AMEND_PARTY_DETAILS
+                AMEND_PARTY_DETAILS,
+                ADD_CASE_NOTE
             )
         ),
         entry(
             CLAIM_DISMISSED_PAST_CLAIM_NOTIFICATION_DEADLINE.fullName(),
-            List.of(CASE_PROCEEDS_IN_CASEMAN)
+            List.of(
+                CASE_PROCEEDS_IN_CASEMAN,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             CLAIM_DISMISSED_PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE.fullName(),
-            List.of(CASE_PROCEEDS_IN_CASEMAN)
+            List.of(
+                CASE_PROCEEDS_IN_CASEMAN,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             PAST_APPLICANT_RESPONSE_DEADLINE_AWAITING_CAMUNDA.fullName(),
-            List.of(TAKE_CASE_OFFLINE)
+            List.of(
+                TAKE_CASE_OFFLINE,
+                ADD_CASE_NOTE
```

Comment:
I believe in this scenario because we are merely waiting for Camunda to do its job and move the claim to `TAKEN_OFFLINE_PAST_APPLICANT_RESPONSE_DEADLINE` it is already too late. The case should be considered offline and no new notes should be allowed?

Reply:
This one I think you might be right, state would be PROCEEDS_IN_HERITAGE_SYSTEM when taken offline correct?

Reply:
Yes correct 👍